code/game/machinery/computer/camera.dm

```diff
@@ -170,7 +170,7 @@
 
 /obj/machinery/computer/security/attack_ai(mob/user)
 	if(isAI(user))
-		to_chat(user, "<span class='notice'>You realise its kind of stupid to access a camera console when you have the entire camera network at your metaphorical fingertips</span>")
+		to_chat(user, "<span class='notice'>You realize it's kind of stupid to access a camera console when you have the entire camera network at your metaphorical fingertips.</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You realise it's kind of stupid to access a camera console when you have the entire camera network at your metaphorical fingertips.</span>")
```
While code needs to use broken English to keep with BYOND (`color` not `colour`), I believe that written strings can be written in correct English, given that's what we use on the wiki.